### PR TITLE
Tickets/sitcom 1383

### PR DESCRIPTION
**Pull request by Garavena89**
Can you please review this PR about the procedure to restart the MTMount Operation Manager?
Thank you so much

### CODE REVIEW

File: MainTel/MainTel-Non-standard-Operations/MTCS-Non-standard-Procedures/TMA-Non-standard-Procedures/TMA-Restart-MTMount-Operation-Manager-(OM).rst [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+2. The TMA EUI application is operational and MTMount can be enabled without errors.
+
+Ensuring these post-conditions confirms the successful completion of the OM restart procedure and guarantees the continuity of telescope operations.
+
+.. _Title-of-Troubleshooting-Procedure-Contingency:
+
+Contingency
+===========
+
+If the procedure was not successful, report the issue in [relevant Slack channel] and/or activate the :ref:`Out of hours support <Safety-out-of-hours-support>`.
```
Comment (isotuela) [inline MainTel/MainTel-Non-standard-Operations/MTCS-Non-standard-Procedures/TMA-Non-standard-Procedures/TMA-Restart-MTMount-Operation-Manager-(OM).rst:154]: Replace **relevant Slack channel** for the #summit-simonyi.

**Review by isotuela — APPROVED** [review]
This is great! 
Some minor comments. Remember to write in reST one sentence per line for easier diff and reading. 

Thank you!

File: MainTel/MainTel-Non-standard-Operations/MTCS-Non-standard-Procedures/TMA-Non-standard-Procedures/TMA-Restart-MTMount-Operation-Manager-(OM).rst (hunk 1 above)
Comment (Garavena89) [reply MainTel/MainTel-Non-standard-Operations/MTCS-Non-standard-Procedures/TMA-Non-standard-Procedures/TMA-Restart-MTMount-Operation-Manager-(OM).rst:154]: haha I missed this!, my bad, thanks

**Review by PaulinaLSST — COMMENTED** [review]
Revision of the full document

**Comment by Garavena89** [comment]
corrections checked and done, ready to rebase, squash and merge

**Pull request merged**